<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu1">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionSave1"/>
    <addaction name="actionSave2"/>
    <addaction name="actionSave3"/>
   </widget>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="actionLoad1"/>
    <addaction name="actionLoad2"/>
    <addaction name="actionLoad3"/>
   </widget>
   <widget class="QMenu" name="menuPlayerAreas">
    <property name="title">
     <string>Player Areas</string>
    </property>
    <addaction name="actionShow"/>
    <addaction name="actionHide"/>
   </widget>
   <addaction name="menuMenu1"/>
   <addaction name="menuLoad"/>
   <addaction name="menuPlayerAreas"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave1">
   <property name="text">
    <string>Save1</string>
   </property>
  </action>
  <action name="actionSave2">
   <property name="text">
    <string>Save2</string>
   </property>
  </action>
  <action name="actionSave3">
   <property name="text">
    <string>Save3</string>
   </property>
  </action>
  <action name="actionLoad1">
   <property name="text">
    <string>Load1</string>
   </property>
  </action>
  <action name="actionLoad2">
   <property name="text">
    <string>Load2</string>
   </property>
  </action>
  <action name="actionLoad3">
   <property name="text">
    <string>Load3</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
